core/arch/arm/plat-zynqmp/platform_config.h

```diff
@@ -86,6 +86,11 @@
 #error "Unknown platform flavor"
 #endif
 
+#define CSUDMA_BASE		0xFFC80000
+#define CSUDMA_SIZE		0x1000
+#define CSU_BASE		0xFFCA0000
+#define CSU_SIZE		0x5000
```

Comment:
Found following from hardware handoff files:
`APERTURE NAME: CSU, START ADDRESS: FFCA0000, END ADDRESS: FFCAFFFF`

That would make size of 0x10000.